File: 프로그래머스/0/181930. 주사위 게임 2/주사위 게임 2.java
class Solution {
    public int solution(int a, int b, int c) {
        int answer = 0;
        
        if (a == b && b == c && c == a) {
            answer = (a+b+c)*((a*a)+(b*b)+(c*c))*((a*a*a)+(b*b*b)+(c*c*c));
        } else if (a == b && a != c) {
            answer = (a+b+c)*((a*a)+(b*b)+(c*c));
        } else if (b == c && c != a) {
            answer = (a+b+c)*((a*a)+(b*b)+(c*c));
        } else if (c == a && b != a) {
            answer = (a+b+c)*((a*a)+(b*b)+(c*c));
        } else {
            answer = a+b+c;
        }
        return answer;
    }
}

File: 프로그래머스/0/181930. 주사위 게임 2/주사위 게임 2.py
def solution(a, b, c):
    answer = 0
    if (a!=b and b!=c and c!=a):
        answer = a+b+c
    elif (a==b and b==c and c==a):
        answer = (a+b+c)*(a*a+b*b+c*c)*(a*a*a+b*b*b+c*c*c)
    elif (a==b or a==c or b==c):
        answer = (a+b+c)*(a*a+b*b+c*c)
    
        
    return answer